packages/wonder-blocks-modal/util/maybe-get-portal-mounted-modal-host-element.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import {ModalLauncherPortalAttributeName} from "./constants.js";
+
+/**
+ * From a given element, finds its next ancestor that is a modal launcher portal
+ * element.
+ * @param {null | Element | Text} element The element whose ancestors are to be
+ * walked.
+ * @returns {Element | null} The nearest parent modal launcher portal or
+ * `null`.
+ */
+function maybeGetNextAncestorModalLauncherPortal(element: null | Element | Text) {
+    let candidateElement = element ? element.parentElement : null;
```

Comment:
🤷‍♂️ It feels like the logic can be simplified a bit. Perhaps:

`let candidateElement = element && element.parentElement;`

and then later:

`return candidateElement;`

Does it particularly matter if null or undefined is returned?

Reply:
For the first point, definitely can change that. This code originally looked a little different and I didn't golf it down after it hit this stage.

Regarding the second point, let me check the usage. There was a reason why I had it returning null, but that may have since changed. Definitely seems like we should simplify, but I will make sure that anything relying on it returning `null` is fixed too.